### ISSUE TITLE
Issue with Max_wrapped in horizontal-vertical

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#700]
In GitLab by @dbeltrankyl on May 20, 2021, 13:50

Hello @mcastril, @wuruchi 

I found a bug related to this parameter on both versions, 

The max wrapped value doesn't have into account the horizontal list, so if you have

```
DA - DA 
DA - DA 
REDUCE - REDUCE 
```

```
TYPE = horizontal-vertical
MAX_WRAPPED = 2 ( not have much sense but is for illustrating)
```

It will wrap

```
DA - DA 
DA - DA 
```

This is due every time the vertical list is created, it is based in one of the first horizontal jobs, so in AS mind:

```
DA - DA wrapped jobs = 1 instead of 2
DA - DA Wrapped jobs = 2 instead of 4
```

This happens in both versions, I fixed 3.14 but 3.13 it could be a bit intrusive

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 20, 2021, 13:57

Basically, it only limits the max size vertical or horizontal.
So if you have max_wrapped = 4. You can have up to 4x4 jobs inside a wrapper.

**Comment by kinow** [comment]
In GitLab by @mcastril on May 21, 2021, 11:57

So as discussed yesterday, in 3.14.0:

* MAX_WRAPPED should account for the total number of tasks in the wrapper x * y
* MAX_WRAPPED_H should account for the total number of tasks in the horizontal plane (x)
* MAX_WRAPPED_V should account for the total number of tasks in the vertical plane (y)

I used that names as examples but they could differ. Would we allow to specify MAX_WRAPPED_H and MAX_WRAPPED_V in 1D wrappers? Let's say, MAX_WRAPPED_V in the vertical wrapper... I am not sure, but maybe the most simple way is to throw an error if MAX_WRAPPED_H or MAX_WRAPPED_V are specified with 1D wrappers.

**Comment by kinow** [comment]
In GitLab by @mcastril on May 27, 2021, 17:46

@dbeltrankyl I think we need to review the MIN_WRAPPED too, in case that you are not checking that


@f-macchia

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Sep 6, 2021, 13:48

This has been implemented on 3.14.0 ( not yet updated) 

Did some changes to min and jobs specific limits
